composer.json

```diff
@@ -49,6 +49,7 @@
         "php": ">=5.3.3"
     },
     "require-dev": {
+        "hoa/math": "0.15.05.29",
```

Comment:
Absolute non-issue for a dev component I'd say. Also, there is maybe a de
facto standard at best.

On Fri, Nov 20, 2015 at 9:24 PM, Graham Campbell <notifications@github.com>
wrote:

> In composer.json
> <https://github.com/phpseclib/phpseclib/pull/782#discussion_r45514096>:
>
> > @@ -49,6 +49,7 @@
> >          "php": ">=5.3.3"
> >      },
> >      "require-dev": {
> > +        "hoa/math": "0.15.05.29",
>
> Shame we have to depend on a package with non-standard versioning
> conventions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phpseclib/phpseclib/pull/782/files#r45514096>.
>